packages.Rmd

```diff
@@ -76,6 +76,17 @@ knitr::opts_chunk$set(echo = FALSE)
 </th>
 </tr>
 
+<tr>
+<th style="text-align: center">
+[`geotargets`](https://github.com/njtierney/geotargets)
+</th>
+<th style="font-weight:normal;">
+<br>
+[![](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip) Geospatial packages like `terra` and `sf` don't work well with `targets` without extensive customization. The `geotargets` package is a targets workflow framework for geospatial data. It currently supports raster and vector formats created by `terra` and is progressively adding support for `sf`, and `stars`. For example, `tar_terra_rast()` allows you to work with `terra` `SpatRaster` objects.
```

Comment:
Instead of:

> Geospatial packages like `terra` and `sf` don't work well with `targets` without extensive customization. The `geotargets` package is a targets workflow framework for geospatial data.

How about this:

> The `geotargets` package is a `targets` workflow framework for geospatial data, focusing on the integration between `targets` and geospatial storage formats.

Reply:
Sounds great to me!